debugging/2.0/_index.md

```diff
@@ -10,13 +10,14 @@ the original source (for example, source file and line number references).
 
 * [Jakarta Debugging Support for Other Languages 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jsp/releases/2.0-debugging-support-other-languages)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-spec-2.0-RC2.pdf) (PDF)
-* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-spec-2.0-RC2.html) (HTML)
+* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-spec-2.0.pdf) (PDF)
+* [Jakarta Debugging Support for Other Languages 2.0 Specification Document](./jdsol-spec-2.0.html) (HTML)
+* [Jakarta Debugging Support for Other Languages TCK](https://download.eclipse.org/jakartaee/debugging/2.0/jakarta-debugging-tck-2.0.0.zip)([sig](),[sha](),[pub]())
```

Comment:
The urls for the sig, sha, and pub files need to be filled in.  Not an issue for the ballot, but this needs to be cleaned up before this PR is merged.

Reply:
I've added these since we know what they will be in advance.